.idea/.name

```diff
@@ -1 +1 @@
-sample-respoke-android
+sample-respoke-android
```

Comment:
Android Studio is doing these things, not I. The new version changed a fair bit in these config files.